frontend/src/util/validators.js

```diff
@@ -91,6 +91,47 @@ export function validateDOB(dateOfBirth) {
   return true
 }
 
+/**
+ * Validates that the Date is not earlier than '1900-01-01' .
+ */
+export function validateMinimumDate(dateInput) {
+  if (!helpers.req(dateInput)) {
+    return true
+  }
+  if (!dayjs(dateInput).isAfter(dayjs('1900-01-01'))) {
```

Comment:
Should this be isSameOrAfter? Because 1900-01-01 is a valid entry.

Reply:
isSameOrAfter() function not seems working . Used isBefore to allow 1900-01-01